storagenode/storagenodedb/infodb.go

```diff
@@ -263,14 +263,10 @@ func (db *InfoDB) Migration() *migrate.Migration {
 				Description: "Optimize index usage.",
 				Version:     10,
 				Action: migrate.SQL{
-					`DROP INDEX idx_used_serial`,
 					`DROP INDEX idx_pieceinfo_expiration`,
-					`DROP INDEX idx_bandwidth_usage_created`,
 					`DROP INDEX idx_order_archive_satellite`,
 					`DROP INDEX idx_order_archive_status`,
-					`CREATE INDEX idx_used_serial ON used_serial(datetime(expiration))`,
-					`CREATE INDEX idx_pieceinfo_expiration ON pieceinfo(datetime(piece_expiration)) WHERE piece_expiration IS NOT NULL`,
-					`CREATE INDEX idx_bandwidth_usage_created ON bandwidth_usage(datetime(created_at))`,
+					`CREATE INDEX idx_pieceinfo_expiration ON pieceinfo(piece_expiration) WHERE piece_expiration IS NOT NULL`,
 				},
```

Comment:
shouldn't we have a migration to convert all timestamps to UTC, in case they already weren't?

Reply:
We're dropping the databases during the wipe.